### PR TITLE
Kick-off release 1.20.2

### PR DESCRIPTION
**Pull request by oschaaf**
Signed-off-by: Otto van der Schaaf <ovanders@redhat.com>

### CODE REVIEW

**Comment by RyanTheOptimist** [comment]
This looks reasonable to me. @yanavlasov does this seem like the right approach?

File: docs/root/version_history/current.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-1.20.1 (November 30, 2021)
```
Comment (yanavlasov) [inline docs/root/version_history/current.rst:1]: This file needs to copied into v1.20.1.rst before modification.

**Review by yanavlasov — CHANGES_REQUESTED** [review]
/wait

File: docs/root/version_history/current.rst (hunk 1 above)
Comment (oschaaf) [reply docs/root/version_history/current.rst:1]: Done in 76c6b7939eda59f77e9f9246cc5c1d2009f166ea

**Review by yanavlasov — APPROVED** [review]

**Comment by yanavlasov** [comment]
Looks like some environment issue with the sanitizer builds. Checking with other maintainers for solutions

**Pull request merged**